include/odp/api/spec/packet.h

```diff
@@ -2150,6 +2150,42 @@ uint32_t odp_packet_payload_offset(odp_packet_t pkt);
  */
 int odp_packet_payload_offset_set(odp_packet_t pkt, uint32_t offset);
 
+/** Packet Tx completion mode */
+typedef enum odp_packet_tx_compl_mode_t {
+	/** Packet Tx completion event is disabled. */
+	ODP_PACKET_TX_COMPL_DISABLED,
+	/** Packet Tx completion event is enabled on all transmitted or dropped pkts */
+	ODP_PACKET_TX_COMPL_ALL_XMIT_OR_DROP,
```

Comment:
Maybe this flag could be simply: ODP_PACKET_TX_COMPL_ALL. Then the comment could be that "..completion event is sent for all packets (both for transmitted and dropped)".

Later on, there could be than ODP_PACKET_TX_COMPL_TX, ODP_PACKET_TX_COMPL_DROP, ...